dask/array/core.py

```diff
@@ -1610,6 +1610,12 @@ def vnorm(self, ord=None, axis=None, keepdims=False, split_every=None,
               out=None):
         """ Vector norm """
         from .reductions import vnorm
+
+        warnings.warn(
+            "Please use `dask.array.linalg.norm` instead.",
+            DeprecationWarning
+        )
```

Comment:
Since this just calls `vnorm` (which is already deprecated), no need to have a warning here as well.